src/Task/SecurityCheckerEnlightn.php

```diff
@@ -24,10 +24,12 @@ public static function getConfigurableOptions(): ConfigOptionsResolver
         $resolver->setDefaults([
             'lockfile' => './composer.lock',
             'run_always' => false,
+            'allow_list' => []
```

Comment:
The method `provideConfigurableOptions` in the test should be altered to cover this newly added default.